gc3libs/__init__.py

```diff
@@ -1818,19 +1818,21 @@ def info():
           u'a second message ...'
 
         """
+        return locals()
```

Comment:
Close, but no cigar :-)  The `return locals()` is part of the interface needed by `defproperty`, it should not appear in modernized code.  You should have something like this:

```python
@property
def info(self):
  """A simplified interface for reading/writing entries into history."""
  # copy here definition of `fget` from defproperty stanza  
  return self.history.last()

@info.setter
def info(self, value):
  # ... definition of `fset` from the `defproperty` stanza
```

This applies to all of the `defproperty` changes below.

Reply:
Hah OK, thanks for the review, can fix these. Wasn't really sure what returning locals was all about to be honest :)